cyber/base/unbounded_queue.h

```diff
@@ -71,9 +71,7 @@ class UnboundedQueue {
     do {
       head_next = old_head->next;
 
-      if (head_next == nullptr) {
-        return false;
-      }
+      RETURN_VAL_IF_NULL(head_next, false);
```

Comment:
Personally I'm against macros. See a long story here:
https://github.com/ApolloAuto/apollo/pull/7552#issuecomment-478438061

I'll leave the decision to Cyber owners.

Reply:
Don't use such macros here, especially it requires extra dependency.

Reply:
Btw, please don't use framework as the commits title tag, use cyber instead.

Reply:
thanks @xiaoxq and @GoLancer. revert all changes in directory `cyber/base`